coremltools/converters/mil/mil/ops/defs/iOS15/elementwise_unary.py

```diff
@@ -887,9 +887,12 @@ def get_cast_value(input_var, dtype_val):
             )
 
         if input_var.val is None:
-            if input_var.sym_val is not None and not is_symbolic(input_var.sym_val) and len(input_var.sym_val.shape) == 1:
-                result = [np.array(val).astype(dtype=type_map[dtype_val]).item() if not is_symbolic(val) else val for val in input_var.sym_val]
-                return np.array(result)
+            if input_var.sym_val is not None:
+                if not is_symbolic(input_var.sym_val) and len(input_var.sym_val.shape) == 1:
```

Comment:
This change is not fixing anything for now right?
At least for this particular issue,
it is doing nothing.
Should we remove it?
If we want to keep it, we should at least at a test for it

Reply:
You're right we don't need this. I've reverted it.